backend/src/main/java/aiversum/backend/rest/dto/UserConfigDto.java

```diff
@@ -3,6 +3,9 @@
 public record UserConfigDto(
         String imageSize,
         String imageStyle,
-        String apiKey
+        String apiKey,
```

Comment:
We should wrap it in for every different type of provider: ollama, vertex ai, open ai